Creating a contact form with william and gardimy

tomasesquivelgc opened the pull request:
## In this branch 

my partners (@Gardimy and @willococa) and I have created this project's contact section. This includes:
  - h2 title for the section
  - a form element that includes name, email and message with a submit button
  - a social media bar at the bottom of the section

tomasesquivelgc committed:
created contact-section

add h2 as title
add form
add li elements with inputs

willococa committed:
added styles to contact form

Gardimy committed:
Update contact form

tomasesquivelgc committed:
changes to contact section

change size units from px to em
center submit button
add contact-media ul with dark social media icons

tomasesquivelgc committed:
fix syntax errors

OyePriscilla commented on index.html:
```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
   </section>
+  <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
```
- [x] Please kindly take note at the moment that the form is not submitting to the url because you did not provide a valid `form_id`. I think you will have to [register with formspree](https://formspree.io/register) before you can have access to the `form_id`. ℹ️

OyePriscilla commented on index.html:
```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
+      <li><a href="#"><img src="./Normal_Button/dark-icon-angelist.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/Dark-icon-twitter.png"></a></li>
+      <li><a href="#"><img src="./Normal_Button/dark-icon-medium.png"></a></li>
+    </ul>
```
- [x] Please kindly add this shape according to the given template. 👍 

##Figma design
<img width="282" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/228354721-6bed09c6-582a-4f09-916f-19d1c1b12c06.png">

OyePriscilla commented on index.html:
```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
+          <input type="email" name="email" placeholder="Email address"  required/>
+        </li>
+        <li>
+          <textarea id="msg" name="user_message" maxlength="500" placeholder="Write me something..." required></textarea>
```
- [x] Please ensure to give this input element padding-left and padding-top according to the design. 👍

OyePriscilla commented on index.html:
```diff
@@ -148,5 +148,31 @@ <h2>About me</h2>
+  <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
```
- [x] Kindly layout the elements in this container with flexbox, it makes our work easier to design and build responsive web pages without having to use tricky hacks. ℹ️

OyePriscilla reviewed (changes requested):
Hi @tomasesquivelgc, @willococa and @Gardimy 👍 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close](https://user-images.githubusercontent.com/103386183/196423863-0f522555-1b03-455e-a452-b822cdcfc398.gif)

## Highlights
Create forms with HTML5 validations ✔️
All fields have the correct type ✔️ 
Implement the button interactions ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [optional] Please, since you're not adding the "Run tests" and "Deployment" sections of the README file at the moment, kindly remove it from the table of content as well. 👍  **Note: Don't forget to add it when required in the future milestones.**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
changes as per review request

contact-form action change to work
bottom image of contact-section add
give contact-form input elements padding
contact-form now display flex

victorokoroji reviewed (approved):
Hi @tomasesquivelgc, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Form submitted successfully ✅ 
- The README is well documented ✅ 
- Good UI design ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Gardimy commented:
Congrats bro! keep going


On Tue, Mar 28, 2023 at 6:00 PM Victor Ebube Okoroji <
***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Hi @tomasesquivelgc <https://github.com/tomasesquivelgc>,
> ✔️ Status: APPROVED 🏆 🎉 🎊
>
> You have done a great job on this project, keep it up 👍 🥇. Your project
> is complete! There is nothing else to say other than... it's time to merge
> it [image: :shipit:]
> congratulations! 🎉💯🌟
> Top highlight 🎯
>
>    - The Pull Request is following the GitHub Flow ✅
>    - The PR title and description are professional and descriptive ✅
>    - Form submitted successfully ✅
>    - The README is well documented ✅
>    - Good UI design ✅
>
>
> <https://camo.githubusercontent.com/5d2ef7789f70f7829b4c278f18e77bc67da5412b06f77ddc5b0c1b0cff45702a/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tomasesquivelgc/Portfolio_Mobile_First/pull/6#pullrequestreview-1361965963>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2SPKROSQOGRRCA4THSKVVLW6NNPTANCNFSM6AAAAAAWLABDAA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>
